src/build-data/policy/bsi.txt

```diff
@@ -182,5 +195,6 @@ x919_mac
 
 # passhash
 bcrypt
+passhash9
```

Comment:
Do they explicitly prohibit PBKDF2? If no, let's just leave it to the user.

Reply:
The guideline does not explicitly prohibit anything as far as I know. They only recommend the most secure stuff. In this case [they only recommend Argon2](https://www.bsi.bund.de/SharedDocs/Downloads/EN/BSI/Publications/TechGuidelines/TG02102/BSI-TR-02102-1.pdf?__blob=publicationFile&v=6#page=72): 
> If the use of a a cryptographic hardware token for password-based key derivation is not possible,
the hash function Argon2id [14] should be used.

I would suggest the same as for the cipher modes: If there is no important dependency we should stay consistent and add or remove both. 


Additional Note: [Wikipedia quote](https://en.wikipedia.org/wiki/PBKDF2#Alternatives_to_PBKDF2):

> One weakness of PBKDF2 is that while its number of iterations can be adjusted to make it take an arbitrarily large amount of computing time, it can be implemented with a small circuit and very little RAM, which makes brute-force attacks using [application-specific integrated circuits](https://en.wikipedia.org/wiki/Application-specific_integrated_circuit) or [graphics processing units](https://en.wikipedia.org/wiki/Graphics_processing_unit) relatively cheap.

Not critical, but the reason to create a competition to create a better algorithm (which was Argon2).

Reply:
For now, I will remove the prohibition as suggested.